x-pack/plugins/search_playground/public/components/sources_panel/sources_panel_for_start_chat.tsx

```diff
@@ -67,6 +81,19 @@ export const SourcesPanelForStartChat: React.FC = () => {
         </EuiFlexItem>
       )}
 
+      {sourceFieldErrorMessage && (
+        <EuiCallOut color="warning" iconType="warning">
+          <p>
+            {i18n.translate('xpack.searchPlayground.emptyPrompts.sources.warningCallout', {
+              defaultMessage: 'No source fields found for {errorMessage}',
```

Comment:
cc: @leemthompo for the copy

Reply:
@leemthompo : Merging this PR for now, however please comment in the PR if you want this text to reflect anything different. We will tackle this from another PR.

Reply:
@Samiul-TheSoccerFan this looks good, but I'd need to know the range of values `sourceFieldErrorMessage` returns to know if it makes sense in all scenarios :)

Reply:
@leemthompo The `sourceFieldErrorMessage` will reflect the name of the indices which does not have any source fields associated.